app/controllers/grant_submissions/submissions/submission_applicants_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+module GrantSubmissions
+  module Submissions
+    class SubmissionApplicantsController < GrantBaseController
+      skip_after_action :verify_policy_scoped, only: %i[index]
+      before_action     :set_submission_and_grant
+
+      def index
+        authorize @submission, :edit?
+        @submission_applicants = @submission.submission_applicants.includes(:applicant).order("#{User.table_name}.last_name, #{User.table_name}.first_name")
+      end
+
+      def create
+        authorize @submission, :edit?
+
+        email   = submission_applicant_params[:applicant_email].downcase.strip
+        user = User.find_by(email: email)
+
+        if user.nil?
+          flash[:alert] = "Could not find a user with the email: #{email}."
+          # #{helpers.link_to 'Invite them to be a reviewer', invite_grant_reviewers_path(@grant, email: email), method: :post, data: { confirm: "An email will be sent to #{email}. You will be notified when they accept or opt out."} }"
+        else
+          submission_applicant = GrantSubmission::SubmissionApplicant.create(submission: @submission, applicant: user)
+          if submission_applicant.errors.none?
+            flash[:success] = "Added #{helpers.full_name(submission_applicant.applicant)} as applicant."
+          else
+            flash[:alert]   = submission_applicant.errors.full_messages
+          end
+
+        end
+
+        redirect_to grant_submission_submission_applicants_path(@grant, @submission)
+      end
+
+      def destroy
+        authorize @submission, :edit?
+        applicant = GrantSubmission::SubmissionApplicant.find(params[:id])
+
+        if applicant.nil?
+          flash[:alert] = 'Applicant could not be found.'
+        else
+          if applicant.destroy
+            flash[:notice] = 'Applicant was deleted.'
```

Comment:
Seems harsh. Change to something like "#{full_name(applicant)} is no longer an applicant on #{submission.name}" 

Should the removed applicant be notified?

Reply:
Changed the text of the alert and added new specs for the alert.

I added a new ticket for the notification email in #848.